test/cri-containerd/runpodsandbox_test.go

```diff
@@ -1137,6 +1139,77 @@ func Test_RunPodSandbox_Mount_SandboxDir_LCOW(t *testing.T) {
 	//TODO: Parse the output of the exec command to make sure the uvm mount was successful
 }
 
+func Test_RunPodSandbox_CPUGroup(t *testing.T) {
+	ctx := context.Background()
+	presentID := "FA22A12C-36B3-486D-A3E9-BC526C2B450B"
```

Comment:
Any reason for this ID in particular?

Reply:
Just a fake ID for testing. It doesn't matter what this value is.

Reply:
Sweet, thanks for clarifying